Makefile

```diff
@@ -56,7 +56,7 @@ help:
 	@echo -e "When building for the board, you can specify custom define values for config.h."
 	@echo -e "See Config/Inc/config.h for all values."
 	@echo -e "Specify with the following ${PURPLE}format${NC}(quotes ARE needed):"
-	@echo -e "		${ORANGE}make ${BLUE}DEFINES=${GREEN}\"-D VARIABLE=VALUE VARIABLE=VALUE\"\n"
+	@echo -e "		${ORANGE}make ${BLUE}DEFINES=${GREEN}\"-D VARIABLE=VALUE -D VARIABLE=VALUE\"\n"
```

Comment:
Once this gets pushed I can update the help print in my makefile branch, its DEFINES=<name>=VALUE/<name>=VALUE etc